src/lib/parameters.ml

```diff
@@ -0,0 +1,102 @@
+open Nonstd
+module String = Sosa.Native_string
+
+
+type t = {
+  (* MHC Alleles which take precedence over those generated by Seq2HLA. *)
+  mhc_alleles: string list option;
+  mouse_run: bool [@default false];
+  with_topiary: bool [@default false];
+  with_seq2hla: bool [@default false];
+  with_mutect2: bool [@default false];
+  with_varscan: bool [@default false];
+  with_somaticsniper: bool [@default false];
+  with_optitype_normal: bool [@default false];
+  with_optitype_tumor: bool [@default false];
+  with_optitype_rna: bool [@default false];
+  email_options: Qc.EDSL.email_options option;
+  bedfile: string option [@default None];
+  experiment_name: string [@main];
+        reference_build: string;
+        normal_inputs: Biokepi.EDSL.Library.Input.t list; (* 1+ items *)
+        tumor_inputs: Biokepi.EDSL.Library.Input.t list;  (* 1+ items *)
+        rna_inputs: Biokepi.EDSL.Library.Input.t list option;    (* 0+ items *)
+        picard_java_max_heap: string option;
+        igv_url_server_prefix: string option;
+        vaxrank_include_mismatches_after_variant: bool [@default false];
+        leave_input_bams_alone: bool [@default false];
+        use_bwa_mem_opt: bool [@default true];
```

Comment:
indentation
(https://github.com/OCamlPro/ocp-indent/issues/226, my usual hack-around is to put the `[@main]` field at the end of the record)